### ISSUE TITLE
Investigate purchasing Intel bonuses like the network tower mission

### ISSUE THREAD

**Issue by MalucoMarinero** [MalucoMarinero/XCOM2GuerillaWarfare#12]
The Advent Tower has this cool gimmick where you can spend Intel to get tactical advantages. It'd be interesting to use that in general missions to confer tactical bonuses at a price.

**Comment by deuspingu** [comment]
There is already a mod for this on Steam Workshop, called Intel Purchasing or something like that.

**Comment by MalucoMarinero** [comment]
Yep, that's RealityMachina's http://steamcommunity.com/sharedfiles/filedetails/?id=655190779&searchtext=Intel

**Comment by MalucoMarinero** [comment]
this won't be followed til after the playtesting release.
